content/doc/book/using/aborting-a-build

```diff
@@ -0,0 +1,62 @@
+---
+layout: section
+title: Aborting a build
+---
+
+= Aborting a build
+
+When you click that [x] icon from the UI, the following things happen:
+
+. Browser sends a request to the server.
+. The server interrupts (via Thread.interrupt()) the thread (AKA
+executor thread) that is responsible for carrying out a build.
+. The server returns
+
+At this point, your browser is back, but the actual abort process
+happens from here asynchronously.
+
+. The thread gets the interrupt signal. How quickly this happens depends
+on what the executor is doing at the time of the interruption.
+Specifically, an executor thread can only be interrupted in
+"interruption points" due to the Java design.
+* Waiting for a completion of a child process (for example, maybe the
+build is running Ant) is an interruption point. That means if the
+executor was doing that, it gets interrupted instantaneously.
+* Waiting for a computation on a slave is an interruption point.
+* Waiting for file or network I/O is not an interruption point. This
+often causes the problem where a build appears to be un-abortable. For
+example, checking out a Subversion repository falls in this category.
+* Normal computation is also not an interruption point.
+. The executor performs a clean up operation. This depends on what it
+was doing by the time it noticed the interruption.
+* If it was waiting for a completion of a child process, Jenkins will
+search for all the descendant processes and kill them all. On Unix, this
+is done through `+java.lang.UnixProcess.destroyProcess+`, which sends
+http://en.wikipedia.org/wiki/SIGTERM[SIGTERM] on Sun's JREs. On Windows,
```

Comment:
SIGTERM behavior is Unix-based behavior, not limited to Sun JRE's (especially since Sun JRE's don't exist anymore).
```suggestion
http://en.wikipedia.org/wiki/SIGTERM[SIGTERM] on Unix-based JDK implementations. On Windows,
```